### DISCUSSION TITLE
C128 VDC support

### DISCUSSION THREAD

**Discussion by xlar54** [frntc/Sidekick64#32] [Q&A] [answered]
How are you providing VDC support in Ultimax mode?  I see that code is generated.  Is it then executed periodically to mirror the 40 col screen?

**Comment by frntc** [comment, marked as answer]
Using code to mirror the 40-column screen is what I tried first, but that's horribly slow. The Sidekick generates speed code for both 40 and 80 column screen updates. It's not running in Ultimax mode ;-)
